modules/aurora-postgres-resources/default.auto.tfvars

```diff
@@ -0,0 +1,5 @@
+# This file is included by default in terraform plans
+
+enabled = false
+
+name = "pg"
```

Comment:
let's move `name` to YAML config (we had issues in Spacelift when we define vars in many places)